source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,245 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
 Lastest Mattermost Releases:
+- [Release v6.0 - Feature Release](#release-v6.0-feature-release)
 - [Release v5.39 - Quality Release](#release-v5-39-quality-release)
 - [Release v5.38 - Feature Release](#release-v5-38-feature-release)
 - [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
 - [Release v5.36 - Feature Release](#release-v5-36-feature-release)
 - [Release v5.35 - Feature Release](#release-v5-35-feature-release)
-- [Release v5.31 - ESR](#release-v5-31-esr)
+
+## Release v6.0 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2021-10-13**
+
+### Deprecations
+
+1. [Legacy Command Line Tools](https://docs.mattermost.com/manage/command-line-tools.html). All commands have been fully replaced by [mmctl](https://docs.mattermost.com/manage/mmctl-command-line-tool.html) and new commands have been added over the last few months, making this tool a full and robust replacement. 
+
+2. [Slack Import via the web app](https://docs.mattermost.com/onboard/migrating-to-mattermost.html#migrating-from-slack-using-the-mattermost-web-app). The Slack import tool accessible via the Team Setting menu has been replaced by the [mmetl](https://docs.mattermost.com/onboard/migrating-to-mattermost.html#migrating-from-slack-using-the-mattermost-mmetl-tool-and-bulk-import) tool that is much more comprehensive for the types of data it can assist in uploading. 
+
+3. MySQL versions below 5.7.7. Minimum support is now for 5.7.12. This version introduced a native JSON data type that lets us improve performance and scalability of several database fields (most notably Users and Posts props). Additionally, version 5.6 (our current minimum version) reached [EOL in February 2021](https://www.mysql.com/support/eol-notice.html).
```

Comment:
Technically, the JSON data type was introduced earlier. But we chose 5.7.12 because of a bug we discovered in earlier versions and maximum supported versions in AWS cloud DBs were 5.7.12 I believe. @streamer45 can correct me if I am wrong.

Reply:
Should we clarify if we support 5.7.7+ or 5.7.12+?

Reply:
`JSON`/`jsonb` data type in DB columns was introduced first in 6.0 . Originally MySQL support was decided for version `5.7.7` since that's the version that introduced JSON as supported datatype. However, we later discovered it had a bug (fixed in `5.7.9`). At that point we decided to support as minimum version something that is commonly used in production systems and settled for `5.7.12` since it's the one used by Amazon Aurora. We should update this to say that we officially support `5.7.12+`.